src/Symfony/Component/Security/Guard/GuardAuthenticatorHandler.php

```diff
@@ -136,4 +135,11 @@ public function handleAuthenticationFailure(AuthenticationException $authenticat
             is_object($response) ? get_class($response) : gettype($response)
         ));
     }
+
+    private function dispatchEvent($eventName, Event $event)
```

Comment:
calling dispatcher with different args is dry :) methodfying a null check, to me, is redundant.